kolibri/plugins/coach/assets/src/views/common/LessonStatus.vue

```diff
@@ -1,35 +1,59 @@
 <template>
 
-  <KPageContainer>
+  <KPageContainer :topMargin="isPrint ? 0 : 24">
     <KGrid gutter="16">
-      <KGridItem class="status-label" :layout12="{ span: 8 }">
-        {{ $tr('visibleToLearnersLabel') }}
-      </KGridItem>
-      <KGridItem class="status-label" :layout12="{ span: 4 }">
-        <KSwitch
-          name="toggle-lesson-visibility"
-          :checked="lesson[activeKey]"
-          :value="lesson[activeKey]"
-          @change="handleToggleVisibility"
-        />
-      </KGridItem>
-      <KGridItem style="font-weight: bold;">
-        {{ coachString('recipientsLabel') }}
-      </KGridItem>
-      <KGridItem>
-        <div>
-          <Recipients
-            :groupNames="groupNames"
-            :hasAssignments="assignments.length > 0"
+      <!-- Class name, for print only -->
+      <div v-if="isPrint" class="status-item">
+        <KGridItem class="status-label" :layout12="layout12Label">
+          {{ coachString('classLabel') }}
+        </KGridItem>
+        <KGridItem :layout12="layout12Value">
+          {{ className }}
+        </KGridItem>
+      </div>
+
+      <!-- Visibility status/switch -->
+      <div v-show="!isPrint" class="status-item visibility-item">
+        <KGridItem class="status-label" :layout12="{ span: 8 }">
+          {{ $tr('visibleToLearnersLabel') }}
+        </KGridItem>
+        <KGridItem :layout12="{ span: 4 }">
+          <KSwitch
+            name="toggle-lesson-visibility"
+            :checked="lesson[activeKey]"
+            :value="lesson[activeKey]"
+            @change="handleToggleVisibility"
           />
-        </div>
-      </KGridItem>
-      <KGridItem class="status-label">
-        {{ coachString('descriptionLabel') }}
-      </KGridItem>
-      <KGridItem>
-        {{ lesson.description || "--" }}
-      </KGridItem>
+        </KGridItem>
+      </div>
+
+      <!-- Recipients -->
+      <div class="status-item">
+        <KGridItem class="status-label" :layout12="layout12Label">
+          {{ coachString('recipientsLabel') }}
+        </KGridItem>
+        <KGridItem :layout12="layout12Value">
+          <div>
+            <Recipients
+              :groupNames="groupNames"
+              :hasAssignments="assignments.length > 0"
+            />
+          </div>
+        </KGridItem>
+      </div>
+
+      <!-- Description -->
+      <div class="status-item">
+        <KGridItem class="status-label" :layout12="layout12Label">
```

Comment:
it might be easier for readability (less scrolling) to just inline the layouts, e.g. 

```html
<KGridItem
  class="status-label"
  :layout12="{ span: this.isPrint ? 3 : 12 };"
>
```